lib/vdsm/virt/thinp.py

```diff
@@ -221,9 +224,13 @@ def on_enospc(self, drive):
 
     # Monitoring volumes.
 
-    def monitor_volumes(self):
+    def monitor_volumes(self, urgent=False):
         """
         Check and extend drives if needed.
+
+        If urgent is True, this is a call trigger by event from libvirt,
```

Comment:
"trigger" -> "triggered"

Reply:
The argument and the docstring is removed in current version.